Source/gmenu.h

```diff
@@ -18,6 +18,56 @@ struct TMenuItem {
 	uint32_t dwFlags;
 	const char *pszStr;
 	void (*fnMenu)(bool);
+
+	[[nodiscard]] bool enabled() const
+	{
+		return (dwFlags & GMENU_ENABLED) != 0;
+	}
+
+	[[nodiscard]] bool isSlider() const
+	{
+		return (dwFlags & GMENU_SLIDER) != 0;
+	}
+
+	[[nodiscard]] uint16_t sliderStep() const
+	{
+		return dwFlags & 0xFFF;
+	}
+
+	void setSliderStep(uint16_t step)
+	{
+		dwFlags &= 0xFFFFF000;
+		dwFlags |= step;
+	}
+
+	[[nodiscard]] uint16_t sliderSteps() const
+	{
+		return (dwFlags & 0xFFF000) >> 12;
+	}
+
+	void setSliderSteps(uint16_t steps)

```

Comment:
`setSliderPosition` maybe? Seems very easy to confuse this with the other function which sets the step/increment value.

Reply:
Not sure what the best name is but this sets the total number of steps (plus  one actually).

The component doesn't keep the difference between min and max, so for example for the `[25, 75]` range it'd be called with `50`.

The "step" granularity is always 1.